### ISSUE TITLE
"Verion 9 of Highlight.js has reached EOL."

### ISSUE THREAD

**Issue by sshwy** [hexojs/hexo-deployer-git#197]
When I am running `npm i hexo-deployer-git --save`, I got following output:

```
Verion 9 of Highlight.js has reached EOL.  It will no longer
  be supported or receive security updates in the future.
  Please upgrade to version 10 or encourage your indirect
  dependencies to do so.

  For more info:
  
  https://github.com/highlightjs/highlight.js/issues/2877
  https://github.com/highlightjs/highlight.js/blob/master/VERSION_10_UPGRADE.md
```

I've read the package-lock.json and it shows that hexo-deployer-git@2.1.0 requires hexo-util@1.0.0, which is different from what in the latest source code in master repo. I think the npm package need updating.

**Comment by Misaka13514** [comment]
# Dependabot cannot update minimist to a non-vulnerable version
The latest possible version that can be installed is `0.0.10` because of the following conflicting dependencies:

```
hexo-deployer-git@2.1.0 requires minimist@~0.0.1 via a transitive dependency on optimist@0.6.1
```
The earliest fixed version is `0.2.1`.

## Remediation

Upgrade **minimist** to version **0.2.1** or later. For example:

```
minimist@^0.2.1:
  version "0.2.1"
```

## Details

CVE-2020-7598

**Comment by stevenjoezhang** [comment]
`hexo-deployer-git` version 3.0.0 has been released, please update the plugin to resolve this issue.

`hexo-deployer-git` 3.0.0 版本已发布，请更新插件来修复此问题。

```
npm i hexo-deployer-git@3.0.0
```
